modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/asset.jag

```diff
@@ -209,14 +209,11 @@ require('/modules/store.js').exec(function (ctx) {
             var result = null;
             var issuerName = null;
             var store = jagg.module("manager").getAPIStoreObj();
-            var isEmailLoginEnabled = store.isEnableEmailUsername();
+            var tenantDomain = Packages.org.wso2.carbon.context.CarbonContext.getThreadLocalCarbonContext().getTenantDomain();
```

Comment:
Not sure is this the correct way to do this. Please get this reviewed by Carbon team.